includes/llms.spam.functions.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * Code related to spam detection and prevention.
+ */
+// Constants. Define these in wp-config.php to override.
+if ( ! defined( 'LLMS_SPAM_ACTION_NUM_LIMIT' ) ) {
```

Comment:
We're using `llms_maybe_define_constant` in other spots but I'm fine to just do this standard pattern, especially if it needs to be defined before the function might be loaded.